shipper.py

```diff
@@ -226,49 +227,108 @@ def shipment_post_new():
                 current_compendium = db['compendia'].find_one({'id': data['compendium_id']})
                 if current_compendium:
-                    # zip all files in dir and submit as zip:
-                    compendium_files = os.path.join(env_compendium_files, data['compendium_id'])
-                    if os.path.isdir(compendium_files):
-                        file_name = '.'.join((str(data['compendium_id']), 'zip'))
-                        if data['recipient'] == 'zenodo':
-                            data['deposition_id'] = zenodo_create_depot(env_repository_zenodo_host, env_repository_zenodo_token)
-                            data['deposition_url'] = ''.join((env_repository_zenodo_host.replace('api', 'deposit/'), data['deposition_id']))
-                            zenodo_add_zip_to_depot(env_repository_zenodo_host, data['deposition_id'], file_name, compendium_files, env_repository_zenodo_token)
-                            # Add metadata that are in compendium in db:
-                            if 'metadata' in current_compendium:
-                                if 'zenodo' in current_compendium['metadata']:
-                                    md = current_compendium['metadata']['zenodo']
-                                    zenodo_add_metadata(env_repository_zenodo_host, data['deposition_id'], md,
-                                                        env_repository_zenodo_token)
-                        elif data['recipient'] == 'eudat':
-                            data['deposition_id'] = eudat_create_depot(env_repository_eudat_host, env_repository_eudat_token)
-                            data['deposition_url'] = ''.join((env_repository_eudat_host.replace('api', 'records/'), data['deposition_id']))
-                            eudat_add_zip_to_depot(env_repository_eudat_host, data['deposition_id'], file_name, compendium_files, env_repository_eudat_token)
-                            # Add metadata that are in compendium in db:
-                            if 'metadata' in current_compendium:
-                                if 'eudat' in current_compendium['metadata']:
-                                    md = current_compendium['metadata']['eudat']
-                                    eudat_update_md(env_repository_eudat_host, data['deposition_id'], md, env_repository_eudat_token)
+                    # Aquire path to files via env var and id:
+                    compendium_files = os.path.normpath(os.path.join(env_compendium_files, data['compendium_id']))
+                    # Determine state of that compendium: Is is a bag or not, zipped, valid, etc:
+                    compendium_state = files_scan_path(compendium_files)
+                    if not compendium_state == 0:
+                        # Case path does not exist:
+                        if compendium_state == 1:
+                            # Case: Is a bagit bag:
+                            logging.getLogger('bagit').setLevel(logging.CRITICAL)
```

Comment:
setting the logger level looks like something that must be done only once, so could go [here](https://github.com/o2r-project/o2r-shipper/pull/18/files#diff-9868a663038d90942c66d7ac74799dcdR44) after the app creation.

Reply:
reasonable! I wanted to silence that library completely because it disrupts the status msgs...
Will be moved.